### ISSUE TITLE
The learning rate of pre-trained autoencoder 

### ISSUE THREAD

**Issue by zy-fang** [eleozzr/desc#38]
In the code comment, it says that the learning rate of the pre-trained autoencoder decays from 0.01, but the actual code decays from 1. Can I know which learning rate it should be？
